packages/codegen-ui/lib/types/form/index.ts

```diff
@@ -72,7 +72,8 @@ export type FormInputType =
   | 'URLField'
   | 'EmailField'
   | 'JSONField'
-  | 'Autocomplete';
+  | 'Autocomplete'
+  | 'FileUploaderField';
```

Comment:
Sounds like we're going w/ `StorageField`

Reply:
renamed on latest commit